### DISCUSSION TITLE
PHP 8.0.3

### DISCUSSION THREAD

**Discussion by peldax** [brefphp/bref#876] [General]
Hello,
I would like to ask you about upgrading to latest patch release of PHP. I understand that maintainting multiple layers is really time consuming and I would like to help. Unfortunately my knowledge of layers is limited and I am not really sure what can I do here, because the layers are probably somehow registered to your account.

I am currently using PHP 8.0 and my lambda came across a really wierd bug with preloading & JIT enabled together (return types ignored & variables having random no-sense values). I would like to create a PHP bug report, but I would like to test on most recent version first - and I am not able to reproduce on my local machine.

Thank you.

**Comment by mnapoli** [comment]
Hi! To get started you can open a pull request like this one: https://github.com/brefphp/bref/pull/802 (bump to the latest version numbers)

I can try to release that this week (indeed for now there's a bit of a manual process involved).

**Reply by peldax** [reply]
https://github.com/brefphp/bref/pull/877
